### PR TITLE
WorkItemDetails

### PR DESCRIPTION
**Pull request by vScottLouvau**
- Arriba 'SUM' merging fix.
- Fix comments in DefaultConfiguration.jsx, so import example.
- Fix help content for recent config/icon changes.
- Add WorkItemDetails in base configuration folder.
- Add Build.Website logic to copy in base configurations.

### CODE REVIEW

File: Arriba/Arriba.Web/jsx/DetailsField.jsx [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
         if (!this.props.value || this.props.value === "N/A" || this.props.value === "Unknown") return null;
-
-        var link = null;
-        if(this.props.url) link = <DetailReportLink url={this.props.url} id={this.props.id} title={this.props["url-title"]} />;
-
+        
         var content = <span dangerouslySetInnerHTML={highlight(this.props.value)} title={this.props.columnName + ':' + stripHighlight(this.props.value)} />;
         if (this.props.bold) content = <b>{content}</b>;
 
+        var link = null;
```
Comment (jeffersonking) [inline Arriba/Arriba.Web/jsx/DetailsField.jsx]: Remove.

File: Arriba/Arriba.Web/jsx/DetailsField.jsx [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
+        if(this.props.url) content = <a target="_blank" href={this.urlReplace(this.urlReplace(this.props.url, "{ID}", this.props.id), "{Value}", this.props.value)} title={this.props.title}>{content}</a>;
+
```
Comment (jeffersonking) [inline Arriba/Arriba.Web/jsx/DetailsField.jsx]: `this.replace(/{ID}/g, encodeURIComponent(this.props.id)).replace(/{Value}/g, encodeURIComponent(this.props.value))`

File: Arriba/Arriba.Web/jsx/DetailsField.jsx [hunk 3]
```diff
@@ -1,21 +1,25 @@
...
         var label = null;
-        if (this.props.label) label = <span>{this.props.label + ": "}</span>;
+        if (this.props.label) label = <span>{this.props.label + " "}</span>;
```
Comment (jeffersonking) [inline Arriba/Arriba.Web/jsx/DetailsField.jsx]: `var  label = this.props.label && <span>{this.props.label + " "}</span>`

File: Arriba/Arriba.Web/jsx/DetailsField.jsx (hunk 1 above)
Comment (vScottLouvau) [inline Arriba/Arriba.Web/jsx/DetailsField.jsx]: fixed

File: Arriba/Arriba.Web/jsx/DetailsField.jsx (hunk 2 above)
Comment (vScottLouvau) [inline Arriba/Arriba.Web/jsx/DetailsField.jsx]: fixed

**Pull request merged**